cli/azd/cmd/init.go

```diff
@@ -165,6 +166,24 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		Title: "Initializing an app to run on Azure (azd init)",
 	})
 
+	// AZD supports having .env at the root of the project directory as the initial environment file.
```

Comment:
Any recommendations for the .env post initializing a template with it? Since it might be lingering? its also excluded in .gitignore?